lib_eio_linux/eio_linux.ml

```diff
@@ -36,6 +36,7 @@ let system_thread = Ctf.mint_id ()
 let wrap_errors path fn =
   try fn () with
   | Unix.Unix_error(Unix.EEXIST, _, _) as ex -> raise @@ Eio.Fs.Already_exists (path, ex)
+  | Unix.Unix_error(Unix.E2BIG, _, _)  as ex -> raise @@ Eio.Fs.File_too_large (path, ex)
```

Comment:
Do we need this? I'd guess that `E2BIG` doesn't always mean "file too large".

Reply:
zapped, no idea what I was thinking